### PR TITLE
Adding awox 33943

### PR DESCRIPTION
**Pull request by wojtekw92**
fixes for #2368

### CODE REVIEW

**Comment by Koenkk** [comment]
Could you share the data/database.db entry of this device with this config?

**Comment by wojtekw92** [comment]
yes
```
{"id":8,"type":"Router","ieeeAddr":"0xa4c138ccabf1af35","nwkAddr":36305,"manufId":4417,"manufName":"AwoX","powerSource":"Mains (single phase)","modelId":"TLSR82xx","epList":[1,3],"endpoints":{"1":{"profId":260,"epId":1,"devId":258,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[6,25],"clusters":{"genBasic":{"attributes":{"modelId":"TLSR82xx","manufacturerName":"AwoX","powerSource":1,"zclVersion":3,"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017"}}},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":49152,"epId":3,"devId":258,"inClusterList":[65360,65361],"outClusterList":[65360,65361],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1615036764282}

```

File: devices.js [hunk 1]
```diff
@@ -13384,6 +13384,23 @@ const devices = [
...
+        fingerprint: [
+            {type: 'Router', manufacturerName: 'AwoX', modelID: 'TLSR82xx', endpoints: [
+                {ID: 1, profileID: 260, deviceID: 258, inputClusters: [0, 3, 4, 5, 6, 8, 768, 4096], outputClusters: [6, 25]},
+                {ID: 3, profileID: 49152, deviceID: 258, inputClusters: [65360, 65361], outputClusters: [65360, 65361]},
+            ]},
+        ],
+        model: '33943',
+        vendor: 'AwoX',
+        description: 'LED RGB & brightness',
+        fromZigbee: [fz.on_off, fz.brightness, fz.level_config, fz.ignore_basic_report],
```
Comment (Koenkk) [inline devices.js:13397]: The `fromZigbee`, `toZigbee` and `exposes` lines can be replaced with: `extend: preset.light_onoff_brightness_colortemp_color(),`

Comment (wojtekw92) [reply devices.js]: ok added :)

**Comment by Koenkk** [comment]
Can you share the data/database.db entry again now?

**Comment by wojtekw92** [comment]
```
{"id":8,"type":"Router","ieeeAddr":"0xa4c138ccabf1af35","nwkAddr":36305,"manufId":4417,"manufName":"AwoX","powerSource":"Mains (single phase)","modelId":"TLSR82xx","epList":[1,3],"endpoints":{"1":{"profId":260,"epId":1,"devId":258,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[6,25],"clusters":{"genBasic":{"attributes":{"modelId":"TLSR82xx","manufacturerName":"AwoX","powerSource":1,"zclVersion":3,"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017"}}},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":49152,"epId":3,"devId":258,"inClusterList":[65360,65361],"outClusterList":[65360,65361],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":0,"stackVersion":2,"hwVersion":0,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1615036764282}

```

**Comment by wojtekw92** [comment]
should I re run zigbee2mqtt on dev to update database.db?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**